### PR TITLE
Fix goimports and add more linting rules

### PR DESCRIPTION
**Pull request by dnkolegov**
This PR fixes the `goimport` format through all the codebase, adds several standard linting rules, and adapts the code according to them.

### CODE REVIEW

File: .golangci.yaml [hunk 1]
```diff
@@ -1,18 +1,28 @@
 linters:
   enable:
+    - gofmt
+    - govet
+    - goimports
+    - misspell
+    - goconst
+    - golint
```
[the remark this answers is not in the document]
Comment (dnkolegov) [reply .golangci.yaml:8]: ok

File: .golangci.yaml [hunk 2]
```diff
@@ -1,18 +1,28 @@
...
     - gocyclo
+    - errcheck
+    - gosec
+    - unconvert
```
[the remark this answers is not in the document]
Comment (dnkolegov) [reply .golangci.yaml:12]: ok

File: .golangci.yaml [hunk 3]
```diff
@@ -1,18 +1,28 @@
...
+    - staticcheck
+    - varcheck
+    - structcheck
+    - deadcode
+    - scopelint
```
[the remark this answers is not in the document]
Comment (dnkolegov) [reply .golangci.yaml:17]: ok

**Review by sergefdrv — APPROVED** [review]
Nice, thanks! :+1:

**Pull request merged**